doc/api/deprecations.md

```diff
@@ -707,7 +707,8 @@ Type: Documentation-only
 
 Using a property named `inspect` on an object to specify a custom inspection
 function for [`util.inspect()`][] is deprecated. Use [`util.inspect.custom`][]
-instead.
+instead. (For backwards compatibility with Node.js prior to version 6.4.0, both
+may be specified.)
```

Comment:
`instead (For backwards compatibility with Node.js prior to version 6.4.0, both may be specified).` ?

Reply:
I think since it's a whole sentence (not just a clause) inside the parentheses the period should be inside.
![image](https://user-images.githubusercontent.com/96947/31046955-efbc8254-a5cf-11e7-9a84-2022c5ca306b.png)
http://www.thepunctuationguide.com/parentheses.html

Reply:
Perhaps the parentheses are unnecessary anyway?